project/OcsBundle.scala

```diff
@@ -355,7 +355,7 @@ trait OcsBundle {
   lazy val bundle_edu_gemini_ags = 
     project.in(file("bundle/edu.gemini.ags")).dependsOn(
       bundle_edu_gemini_shared_skyobject,
-      bundle_edu_gemini_pot,
+      bundle_edu_gemini_pot          % "test->test;compile->compile",
```

Comment:
I assume you mean it took 30 minutes to reload? Did you type `reload` or restart sbt?